include/bio_ik/goal.h

```diff
@@ -67,7 +67,7 @@ class GoalContext
 public:
     GoalContext() {}
     inline const Frame& getLinkFrame(size_t i = 0) const { return tip_link_frames_[goal_link_indices_[i]]; }
-    inline const double getVariablePosition(size_t i = 0) const
+    inline double getVariablePosition(size_t i = 0) const
```

Comment:
This looks like another warning fix that didn't get reverted. Don't get me wrong, it's the correct change to make but lets save this for the later best practices PR unless it's causing immediately problems.